src/tests/__snapshots__/ArtworksContainer.test.js.snap

```diff
@@ -0,0 +1,43 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ArtworksContainer component matches ArtworksContainer component snapshot 1`] = `
+<div
+  className="artworks-container"
+>
+  <div
+    className="d-flex flex-row"
+  >
+    <div
+      className="my-5 col-8"
+    >
+      <img
+        alt="Guernica by Picasso"
+        className="col-10"
+        src="Guernica.jpeg"
+      />
+    </div>
+    <div
+      className="mt-5 col-6"
+    >
+      <h2
+        className="text-start"
+      >
+        Art Institute of Chicago
+      </h2>
+      <p
+        className="text-start"
+      >
+        60 awesome artworks
+      </p>
+    </div>
+  </div>
+  <h6
+    className="text-start my-3 mx-5"
+  >
+    DETAILS BY ARTWORK
+  </h6>
+  <div
+    className="d-flex flex-wrap"
+  />
+</div>
+`;
```

Comment:
- [ ] Please consider working on the `filter` feature let us say you are on the artwork container but you want to access one artwork by maybe the name `Thanksgiving` and access it easily please work on that

Reply:
Hello Delice. The code reviewer previous to yourself had suggested adding a search field on the home page so that a user can easily search for a given artwork by just typing the title. The suggestion was offered as a way to improve the app, and not as a requirement for the project's approval being that the filter function had already been implemented in accordance with Microverse's requirements for the project, and my app varies from the app example in the project specs only in the content that it displays. For me to implement a new function to filter by title, or artist, or other category is quite above the project requirements (as far as I understand them). Please reconsider your request. Thank you.